atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -76,6 +76,17 @@ public void testRenderJava8OptionalsWhenRequested() throws IOException {
                         containsString("import java.util.Optional")));
     }
 
+    @Test
+    public void testIgnoreTableNameLengthFlag() throws IOException {
+        Schema schema = new Schema(
+                "Table",
+                TEST_PACKAGE,
+                Namespace.create("VeryLongNamespaceName"));
+        schema.ignoreTableNameLengthChecks();
+        schema.addTableDefinition("VeryLongTableNameUsedForTesting", getSimpleTableDefinition(TABLE_REF));
+        schema.renderTables(testFolder.getRoot());
+    }
+
```

Comment:
I think we should add a test which verifies that an exception is thrown if we don't specify `ignoreTableNameLengthChecks()`. Maybe even 2 (one for 49-63 and one for 64+).

Also, might be easier for readability if we do `String.join("", Collections,nCopies(n, "x")` where `n` is the number of characters you want in the string.